vembrane/ann_types.py

```diff
@@ -23,6 +23,9 @@ def __eq__(self, other):
     def __bool__(self):
         return False
 
+    def __repr__(self):
+        return "."
```

Comment:
Should it really be `"."`? I'd rather have an empty cell or `None` or `NaN` I guess.